eclair-core/src/main/scala/fr/acinq/eclair/io/Monitoring.scala

```diff
@@ -21,8 +21,10 @@ import kamon.Kamon
 object Monitoring {
 
   object Metrics {
```

Comment:
We could probably also instrument the reconnection tasks? Maybe a counter for successful/failed reconnections?

Reply:
Oh yeah